test/Tests/src/System/File_Spec.enso

```diff
@@ -359,6 +528,22 @@ spec =
                 n.read_text . should_equal "new content"
             [f, bak, n0, n1, n2, n4].each .delete
 
+        Test.specify "should create a backup when writing a dry run file with Context.Output disabled" <|
+            f = transient / "work.txt"
+            f.delete_if_exists
+            f.exists.should_be_false
+            "line 1!".write f on_problems=Report_Error . should_succeed . should_equal f
+
+            r = Context.Output.with_disabled <| "New Content!".write f
+
+            bak = transient / "work.txt.bak"
+            bak.exists.should_be_false
```

Comment:
I'm wondering if this is a good enough test.
It does check that the `work.txt.bak` was not created. But what about the `dry-run-tmpfile.txt.bak`? (name made up, but you get the idea)


Ideally, I'd check that the only new file created is `r`.

However, since we cannot control _where_ the temp files go and the temp directory can be used by other software, I guess we cannot do much better, so I think it's ok to keep it.